kombu/transport/sqlalchemy/models.py

```diff
@@ -37,7 +37,10 @@ def messages(cls):
 class Message(object):
     """The message class."""
 
-    __table_args__ = {'sqlite_autoincrement': True, 'mysql_engine': 'InnoDB'}
+    __table_args__ = (
+        Index('ix_kombu_message_timestamp_id', 'timestamp', 'id'),
```

Comment:
Can you please explain why you chose the two columns together? If it is only for sorting, I guess you only need the timestamp, right?

Reply:
This may be actual only in my case but postgresql planner do this for ~6000000 messages:
with index
![default](https://user-images.githubusercontent.com/16062807/39098835-7628b1ec-4679-11e8-90c4-2305f71aabe6.png)
without index
![default](https://user-images.githubusercontent.com/16062807/39098839-85754dae-4679-11e8-9588-ea924b24b560.png)

Reply:
@mu5h3r thanks for the feedback, yes this makes sense.